client/mainmenu/CPrologEpilogVideo.h

```diff
@@ -19,6 +19,8 @@ class CPrologEpilogVideo : public CWindowObject
 	CampaignScenarioPrologEpilog spe;
 	int positionCounter;
 	int voiceSoundHandle;
+	double voiceDuration;
+	double elapsedTime;
```

Comment:
- perhaps use uint32_t instead and store time in milliseconds? This is how most of our code saves time. Or std::chrono::milliseconds
- either use std::chrono::milliseconds or use more explicit names, e.g. elapsedTimeMilliseconds, elapsedTimeSeconds.